### ISSUE TITLE
Test for WCAG technique G197: Using labels, names, and text alternatives consistently for content that has the same functionality

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#195]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/G197

The purpose of this technique is to help users with cognitive disabilities, blindness and vision loss to understand what will happen when they interact with a function on a Web page. If there are different labels on user interface components (i.e., elements, links, JavaScript objects, etc.) that have the same function, the user will not know that they have encountered a component with the same function and will not know what to expect. This could lead to many unnecessary errors. It is also recommended that this approach to consistent labelling be applied across the Web site.
## Examples
- A Web page has a form field at the top of the page labeled "Search". On the bottom of the page is another form field which provides the same function. It is also labeled "Search."
- A picture of a question mark is used to steer users to sections of the page that provide additional information. Each time the picture of the question mark appears it has the same text alternative "more information."
- A link to the Contact Us page of a Web site has the link text "Contact". At the bottom of the page there is a link that also goes to the Contact Us page. It also has the link text "Contact".

Counterpart Failure: #194 
## Test Procedure
1. Check that each component is associated with text that identifies it (i.e., label, name, or text alternative).
2. Check that this associated text is identical for each user interface component with the same function.
## QUAIL

Unclear if this is a multipage test or singlepage test
